docs/client/python.md

```diff
@@ -60,85 +60,214 @@ The following environment variables are available to use:
 
 If you are using Airflow integration, there are additional [environment variables available](../integrations/airflow/usage.md#environment-variables).
 
-
 #### HTTP transport configuration with environment variables
-For backwards compatibility, the simplest HTTP transport configuration, with only a subset of its config, can be done with environment variables 
+
+For backwards compatibility, the simplest HTTP transport configuration, with only a subset of its config, can be done with environment variables
 (all other transport types are only configurable with YAML file). This setup can be done with the following 
-environment variables:  
-`OPENLINEAGE_URL` (required), `OPENLINEAGE_ENDPOINT` (optional, default: api/v1/lineage), `OPENLINEAGE_API_KEY` (optional).
-
-### Built-in Transport Types
-
-##### HTTP
-- type - string (required)
-- url - string (required)
-- endpoint - string specifying the endpoint to which events are sent. Default: api/v1/lineage (optional)
-- timeout - float specifying a timeout value when sending an event. Default: 5 seconds. (optional)
-- verify - boolean specifying whether the client should verify TLS certificates from the backend. Default: true. (optional)
-- auth - dictionary specifying authentication options. Requires the type property. (optional)
-  - type - string specifying the "api_key" or the fully qualified class name of your TokenProvider. (required if `auth` is provided)
-  - apiKey - string setting the Authentication HTTP header as the Bearer. (required if `type` is `api_key`)
-
-Example:
+environment variables:
+
+- `OPENLINEAGE_URL` (required)
+- `OPENLINEAGE_ENDPOINT` (optional, default: `api/v1/lineage`)
+- `OPENLINEAGE_API_KEY` (optional).
+
+## Built-in Transport Types
+
+### HTTP
```

Comment:
Changed to 3 hashes, to show list of Transport implementations in the menu.

But for Java it didn't appear in the menu. Probably because java_transport.md is imported into configuration.md, and the menu is rendered only for current document, excluding imported ones.